src/GlobalStyle.tsx

```diff
@@ -1,6 +1,48 @@
 import { createGlobalStyle } from 'styled-components';
-import reset from 'styled-reset';
 
 export const GlobalStyle = createGlobalStyle`
-	${reset}
+html, body, div, span, applet, object, iframe,
```

Comment:
font-family 속성을 사용하면 먹혔을 것 같습니다 🙂

[SUIT 깃](https://github.com/sunn-us/SUIT)을 가보니 설명이 되어있었어용!

```suggestion
${reset}

* {
  font-family: ‘SUIT Variable’, sans-serif;
}
```

뭐 이런식으로 함 되지 않을까 싶습니다.

Reply:
이게 styled-reset 에서 명시적으로 스타일을 먹이는 곳이 있는데 그게 * 보다 우선순위가 높아서 font가 무시되네요..

Reply:
넵 제가 늦게 봤군요 이 부분 확인해서 차후 PR때 올려보겠슴다 🙌